src/pretix/control/views/global_settings.py

```diff
@@ -168,12 +168,11 @@ def get_context_data(self, **kwargs):
     def _get_license_for_pkg(self, pkg):
         license, url = None, None
         try:
-            pkgs = pkg_resources.require(pkg)
-            pkg = pkgs[0]
-        except:
+            pkg = pkg_resources.get_distribution(pkg)
+        except pkg_resources.DistributionNotFound:
```

Comment:
The full except was intentional, we had e.g. dev installs crash in some cases because packages weren't correctly installed at all, which was annoying. Maybe this PR fixes all these, but maybe it doesn't, I wouldn't want to take the risk. Otherwise great PR!

Reply:
Alright, I've reverted to the broader exception!